doc/source/rllib/user-guides.rst

```diff
@@ -98,3 +98,12 @@ RLlib Feature Guides
         :type: ref
         :text: How To Work With the RLlib CLI?
         :classes: btn-link btn-block stretched-link
+
+    ---
+    :img-top: /rllib/images/rllib-logo.svg
+
+    +++
+    .. link-button:: rllib-catalogs
+        :type: ref
+        :text: How To Use the RLlib Catalogs?
```

Comment:
```suggestion
        :text: How To Use the RLlib Catalogs
```

Reply:
The construction, "How to Use X" doesn't need a question mark. But I can see the the other cards use question marks, too. If you have time to fix the other occurrences, please do.